### PR TITLE
homepage: our team

### LINKED ISSUE #7: Evacuation Planner
As a refugee, I want a search function,  so that I can look for lists of help, tips and information according to topic and location.

When I open homepage
Then I click "I need help"
Then I choose severity (critical ~ 1h, high ~ today, medium ~ this week, low ~ later)
Then I answer "How can we reach you?" with my phone and email
Then I answer "Where are you now?" selecting city from autocomplete
Then I answer "What help do you need right now?" by selecting a service from the list
Then I see recommended service with (name of contact, phone number, address, website, instructions)
Then I see form to report or review about service
Then I click "Skip for now" and close the page
Then I receive email reminder in 3 days to "Report or Review serice"
Then I click "Share my experience"
Then I write my review and click submit
Then I click "Share to my friends on Facebook" and click "Post" and close the page

[Prototype](https://www.figma.com/proto/VCJKSEBUVXn5xyGShP9LHc/WeHelp-Design?node-id=667%3A599&scaling=scale-down&page-id=10%3A1010&starting-point-node-id=667%3A599&show-proto-sidebar=1)

[See stories](https://github.com/wehelplive/wehelp.live/issues?q=is%3Aopen+is%3Aissue+label%3Aservices)

### PR DESCRIPTION
**Pull request by VOlekh**
## Homepage 2.0

Trello: 
https://trello.com/c/6B3qU5ly/370-add-facebook-icon-on-footer
https://trello.com/c/CVLsHcii/388-rename-opensans-into-sans
https://trello.com/c/T0hM7pHU/387-style-headers-of-sections
https://trello.com/c/7KgRLMSI/316-homepage-section-our-team-3

**Our team section:**
- User scrolls down to our team section. 
- User can see photos of our team members, their names, roles, and social networks.
- User can push the button   _Join Our Team_
- User can't see the _Meet everyone from our team_ button as it is commented due to link absence

## Acceptance Criteria

**Our team section:**
- [x] -  _Join Our Team_ button leads to http://localhost:3000/get-involved
- [x]  - Social networks icons lead to corresponding networks of people
- [x]  - Page made per Figma design
- [x]  - _Meet everyone from our team_ button is out of scope

**Footer section:**

- [x] - Colors and fonts changed per design. Items centered.
- [x] - Order of menu items not altered to be consistent with a header - out of scope
- [x] - All social networks are added including Facebook and lead to corresponding networks of WeHelp

**Start section:**

- [x] - Buttons changed according to _Our team_ section to be consistent.

**Values** and **FAQ** sections:

- [x] - The header is changed according to the _Our Team_ section to be consistent.

**All sections**

- [x] openSans changed to sans

## Demo
https://deploy-preview-56--wehelpvip.netlify.app/

### CODE REVIEW

File: components/block/team.vue [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+          <h2 class="text-3xl mb-10 font-extrabold tracking-tight sm:text-4xl">
+            Our Team
+          </h2>
+        </div>
+        <ul
+          role="list"
+          class="mx-auto grid grid-cols-2 gap-x-4 gap-y-8 sm:grid-cols-4 md:gap-x-6 lg:max-w-5xl lg:gap-x-8 lg:gap-y-12 xl:grid-cols-4"
+        >
+          <li v-for="person in people" :key="person.name">
+            <div class="space-y-4">
```
Comment (razbakov) [inline components/block/team.vue:17]: Please pack each repeatable code as a component. 

Consult with @amanuela97 and check his similar code [here](https://github.com/wehelplive/wehelp.live/pull/37/files#diff-45423db113ce0550e32bac41fa7f598404d49d34195361302ca8f51781d469d1R4)

**Review by razbakov — CHANGES_REQUESTED** [review]
Nice work!

File: components/block/team.vue (hunk 1 above)
Comment (VOlekh) [reply components/block/team.vue:19]: Please take look at a new code, and specify the exact elements on daily.

File: components/layout/footer.vue [hunk 2]
```diff
@@ -3,38 +3,42 @@
...
+          <p
+            class="mx-2 text-zinc-700 font-nunito font-semibold flex justify-center text-base"
+          >
             We help to solve complex problems in the Ukrainian refugee crisis in
             Europe
           </p>
-          <p class="text-grey-600 text-base">#WEHELPUKRAINE</p>
-          <div class="flex space-x-2">
+
+          <div class="flex flex justify-center  space-x-2">
```
Comment (razbakov) [inline components/layout/footer.vue:18]: flex duplicated

File: components/layout/footer.vue [hunk 3]
```diff
@@ -3,38 +3,42 @@
     <h2 id="footer-heading" class="sr-only">Footer</h2>
     <div class="max-w-7xl mx-auto py-18 px-4 sm:px-6 lg:py-16 lg:px-8">
       <div class="xl:grid xl:grid-cols-3 xl:gap-8">
-        <div class="space-y-8 xl:col-span-1">
-          <BrandLogo class="h-24 w-24" />
-          <p class="text-grey-600 text-base">
+        <div class="space-y-8 mx-auto  xl:col-span-1">
```
Comment (razbakov) [inline components/layout/footer.vue:6]:
```suggestion
        <div class="space-y-8 mx-auto xl:col-span-1">
```

File: components/block/team.vue (hunk 1 above)
Comment (razbakov) [reply components/block/team.vue:19]: Just same as Figma components, every repeatable code should be component

**Review by razbakov — CHANGES_REQUESTED** [review]

File: components/block/team.vue (hunk 1 above)
Comment (VOlekh) [reply components/block/team.vue:19]: 1) person card: name, photo, role. 2) buttons ?

File: components/layout/footer.vue (hunk 2 above)
Comment (VOlekh) [reply components/layout/footer.vue:18]: ok, thks

Comment (VOlekh) [reply components/layout/footer.vue:18]: fixed

File: components/block/team.vue (hunk 1 above)
Comment (VOlekh) [reply components/block/team.vue]: fixed

File: components/service/item.vue [hunk 4]
```diff
@@ -44,14 +44,14 @@
...
-    <p class="flex-1 font-openSans text-body text-[#202020] mb-4 line-clamp-2">
+    <p class="flex-1font-sans text-body text-[#202020] mb-4 line-clamp-2">
       {{ description }}
     </p>
 
     <div v-if="url" class="flex flex-row mb-4 items-center">
       <LinkIcon class="h-5 w-5 mr-2 text-grey-500" />
       <a
-        class="text-[#5F6267] flex-1 font-openSans truncate"
+        class="text-[#5F6267] flex-1font-sans truncate"
```
Comment (razbakov) [inline components/service/item.vue:54]:
```suggestion
        class="text-[#5F6267] flex-1 font-sans truncate"
```

File: components/service/item.vue [hunk 5]
```diff
@@ -44,14 +44,14 @@
       </div>
     </div>
 
-    <p class="flex-1 font-openSans text-body text-[#202020] mb-4 line-clamp-2">
+    <p class="flex-1font-sans text-body text-[#202020] mb-4 line-clamp-2">
```
Comment (razbakov) [inline components/service/item.vue:47]:
```suggestion
    <p class="flex-1 font-sans text-body text-[#202020] mb-4 line-clamp-2">
```

File: components/service/item.vue [hunk 6]
```diff
@@ -61,7 +61,7 @@
 
     <div v-if="phone" class="flex flex-row mb-4 items-center">
       <PhoneIcon class="h-5 w-5 mr-2 text-grey-500" />
-      <div class="text-[#5F6267] flex-1 font-openSans">
+      <div class="text-[#5F6267] flex-1font-sans">
```
Comment (razbakov) [inline components/service/item.vue:64]:
```suggestion
      <div class="text-[#5F6267] flex-1 font-sans">
```

**Review by razbakov — CHANGES_REQUESTED** [review]

**Review by amanuela97 — COMMENTED** [review]
FAQ content is not formatted

**Review by amanuela97 — APPROVED** [review]

**Review by razbakov — APPROVED** [review]

**Pull request merged**